docs/docs/pricing/README.md

```diff
@@ -84,63 +99,42 @@ To discuss an Enterprise plan, please contact the Pipedream Sales Team [using th
 
 Pipedream uses a number of terms to describe platform metrics and details of our plans. See the definitions of key terms below.
 
-### Invocations
+### Credit
 
-Pipedream counts an **invocation** each time a workflow or event source is triggered by an incoming event.
+Pipedream counts your usage in terms of _credits_ of total compute time across all workflows in the billing period.
 
-Pipedream increments the count of invocations by one for each incoming event, regardless of the number of steps in your workflow. For example, if you send an HTTP request to a workflow with five steps, Pipedream will count that as one invocation.
+One credit is {{ $site.themeConfig.base_credits_price.seconds }} seconds of compute time at {{ $site.themeConfig.base_credits_price.memory }} megabytes of memory.
 
-Moreover, if you have a workflow triggered by a cron job running once a minute for the entire day, that will incur 1,440 invocations (60 minute \* 24 hours). For cron-triggered workflows or event sources, remember that you can always reduce the frequency to reduce your invocations.
+Regardless of the number of steps in your workflow, if your workflow run runs for {{ $site.themeConfig.base_credits_price.seconds }} or less seconds then that run only costs 1 credit.
```

Comment:
Consider "If a workflow runs for {{ $site.themeConfig.base_credits_price.seconds }} or less at {{ $site.themeConfig.base_credits_price.memory }} MB of memory, that execution is charged one credit." Then I'd combine this sentence with the paragraph above, so that the trivial example flows naturally after the definition.

Reply:
Then maybe follow with that sentence re: steps: "The number of steps in your workflow has no impact on credits."